### PR TITLE
Added code for  Sum of Root To Leaf Binary Numbers

### PR DESCRIPTION
**Pull request by kotwani2883**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

 Sum of Root To Leaf Binary Numbers

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#6439

### CODE REVIEW

File: C-Plus-Plus/ds/RootToLeafSum.cpp [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+int main(){
+	TreeNode* root=newNode(1);
+	root->left=newNode(0);
+	root->right=newNode(1);
+	root->left->left=newNode(0);
+	root->left->right=newNode(1);
+	root->right->left=newNode(0);
+	root->right->right=newNode(1);
```
Comment (siddharth25pandey) [inline C-Plus-Plus/ds/RootToLeafSum.cpp]: Take Dynamic Input from the user

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
Small change is required, rest is fine 💯

**Comment by kotwani2883** [comment]
Made Changes Pls check

**Review by siddharth25pandey — APPROVED** [review]
@kotwani2883 LGTM 💯 Able to merge

**Pull request merged**